lib/modules/manager/gitlabci-include/extract.spec.ts

```diff
@@ -29,6 +31,28 @@ describe('modules/manager/gitlabci-include/extract', () => {
       expect(res?.deps).toHaveLength(3);
     });
 
+    it('extracts multiple embedded include blocks', () => {
+      const res = extractPackageFile(yamlWithTriggerRef);
+      expect(res?.deps).toHaveLength(2);
+      expect(res?.deps).toMatchObject([
+        {
+          currentValue: 'master',
+          datasource: 'gitlab-tags',
+          depName: 'mikebryant/include-source-example',
+        },
+        {
+          currentValue: '1.0.0',
+          datasource: 'gitlab-tags',
+          depName: 'mikebryant/include-source-example',
+        },
+      ]);
+    });
+
+    it('ignores includes without project and file keys', () => {
+      const res = extractPackageFile(yamlWithNonObject);
+      expect(res?.deps).toBeUndefined();
```

Comment:
```suggestion
      expect(res).toBeNull();
```
it should return null if there are no extracted deps